packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -75,6 +76,11 @@ export class Persona {
     return this.accountManager.accounts();
   }
 
+  // TODO implement
+  public async getTxs(): Promise<ReadonlyArray<ProcessedTx>> {
+    return [];
```

Comment:
@webmaster128 Added you as a reviewer just for letting you know I did not have time to implement this method. Is not a big deal, having the signer and the chainIds you can call `searchTx` on each chain. Feel free to implement/rearchitect in the way you can. I just need the `ReadonlyArray<ProcessedTx>`.

Thanks!

Reply:
This can easily be done when #108 is in. The signing server exposes the necessary interface for the transactions